appserver/tests/application/pom.xml

```diff
@@ -122,6 +149,29 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
The original Ant based impl executed wsgen, while in maven it would lead to a chicken - egg problem. So I created WSDL files, used the plugin to generate required classes which were then available for the test.